src/behaviors.js

```diff
@@ -154,10 +145,9 @@ Marionette.Behaviors = (function(Marionette, _) {
 
     // Internal method to build all trigger handlers for a given behavior
     _buildTriggerHandlersForBehavior: function(behavior, i) {
-      var ui = _.extend({}, this._viewUI, _.result(behavior, 'ui'));
       var triggersHash = _.clone(_.result(behavior, 'triggers')) || {};
 
```

Comment:
I'll be honest I don't have a rock solid understanding of it, but its related to this #2144, and seems to be a way to make sure that the UI hash can be updated after the original setup?

It was only implemented for events before.  This change makes it work for triggers.  Didn't work for behavior methods before and won't now.  I'm not sure if thats a problem or not, I don't understand the use case for changing the ui hash after the views been initialized.